copy/copy.go

```diff
@@ -458,18 +484,13 @@ func (ic *imageCopier) copyLayers(ctx context.Context) error {
 	copyGroup := sync.WaitGroup{}
 	copyGroup.Add(numLayers)
 
-	// copySemaphore is used to limit the number of parallel downloads to
-	// avoid malicious images causing troubles and to be nice to servers.
-	var copySemaphore *semaphore.Weighted
-	if ic.c.copyInParallel {
-		copySemaphore = semaphore.NewWeighted(int64(maxParallelDownloads))
-	} else {
-		copySemaphore = semaphore.NewWeighted(int64(1))
-	}
+	// sem reduces the number of parallel goroutines and protects
+	// against malicious images with many layers.
+	sem := semaphore.NewWeighted(24)
```

Comment:
(Make this a top-level constant, please.)

Reply:
Done. I made it consistent with the copySemaphore. I renamed them the fields to be more expressive about their use.